debian/kafka-connect-base/include/etc/confluent/docker/log4j.properties.template

```diff
@@ -5,9 +5,16 @@ log4j.appender.stdout=org.apache.log4j.ConsoleAppender
 log4j.appender.stdout.layout=org.apache.log4j.PatternLayout
 log4j.appender.stdout.layout.ConversionPattern=[%d] %p %m (%c)%n
 
+{% set default_loggers = {
+  'org.reflections': 'ERROR',
```

Comment:
Should this default also include the following?

```
'org.apache.zookeeper': 'ERROR',
'org.I0Itec.zkclientr': 'ERROR',
```

Reply:
Fixed